# EXCEPTION CASE 3
try:
    pass
    raise
except Exception:
    print("foovar-->",e)
    pass
else:
    print("foo")
    pass
